cryptofeed/backends/quasar.py

```diff
@@ -62,3 +73,6 @@ def format(self, data: dict):
         data['stop'] = datetime.utcfromtimestamp(data['stop'])
         data['closed'] = int(data['closed'])
         return data
+    
+    def _create_query(self):
+        self.query = f'CREATE TABLE "{self.table}" (exchange STRING, symbol SYMBOL(currency), start TIMESTAMP, stop TIMESTAMP, interval STRING, trades STRING, open DOUBLE, close DOUBLE, high DOUBLE, low DOUBLE, volume DOUBLE, closed INT64) SHARD_SIZE = {self.shard_size}'
```

Comment:
Why did you call the symbol table for the column `symbol` `currency`? I also think exchange should be a symbol.

```
... (exchange SYMBOL(exchange), symbol SYMBOL(symbol), ....
```

Reply:
Right, this naming doesn't make any sense